README.md

```diff
@@ -186,10 +187,20 @@ Default region name [None]:
 Default output format [None]:
 ```
 
+For this quickstart, we assume you are replicating to an S3 server running
+locally. So let's update the destination to use the host `127.0.0.1` and the
+default S3 port `8000`:
+
+```
+sed 's/const destHost = .*/const destHost = "127.0.0.1:8000"/' \
+~/replication/backbeat/bin/replication.js > \
+~/replication/backbeat/bin/localReplication.js \
+```
+
 Set up replication on your buckets:
 
 ```
-node ~/replication/backbeat/bin/setupReplication.js setup \
+node ~/replication/backbeat/bin/localReplication.js setup \
 --source-bucket source-bucket \
 --source-profile backbeatuser \
```

Comment:
'backbeatuser' should be changed to add profiles 'bart' for source and 'lisa' for destination